core/src/main/cljc/jarl/builtins/time.cljc

```diff
@@ -0,0 +1,179 @@
+(ns jarl.builtins.time
+  (:require [clojure.string :as str]
+            [tick.core :as t]
+            [tick.locale-en-us]
+            [tick.timezone]
+            [jarl.exceptions :as errors]
+            [jarl.utils :refer [instant->ns ns->instant]]))
+
+(def iso-zoned-date-time-fmt (t/predefined-formatters :iso-zoned-date-time))
+(def iso-local-date-time-fmt (t/predefined-formatters :iso-local-date-time))
+
+; Courtesy of https://github.com/newm4n/go-dfe
+(def ^:private go->java-map
+  {; Go          Java
+   "January"   "MMMM"
+   "Jan"       "MMM"
+   "1"         "M"
+   "01"        "MM"
+   "Monday"    "EEEE"
+   "Mon"       "EEE"
+   "2"         "d"
+   "_2"        "_d"
+   "02"        "dd"
+   "15"        "HH"
+   "3"         "K"
+   "03"        "KK"
+   "4"         "m"
+   "04"        "mm"
+   "5"         "s"
+   "05"        "ss"
+   "2006"      "yyyy"
+   "06"        "yy"
+   "PM"        "aa"
+   "pm"        "aa"
+   "MST"       "Z"
+   "Z0700"     "'Z'XX"
+   "Z070000"   "'Z'XX"
+   "Z07"       "'Z'X"
+   "Z07:00"    "'Z'XXX"
+   "Z07:00:00" "'Z'XXX"
+   "-0700"     "XX"
+   "-070000"   "'Z'XX"
+   "-07"       "X"
+   "-07:00"    "XXX"
+   "-07:00:00" "XXX"
+   "999999999" "SSS"})
```

Comment:
Man, I despise this golang time template format .. 

I'd be down for us opening up here, and additionally support the more sane format used internally.